docs/src/references/tedge-write.md

```diff
@@ -0,0 +1,73 @@
+---
+title: tedge-write
+tags: [Reference, Configuration, CLI]
+description: Description of tedge-write component
+---
+
+**tedge-write** is a helper thin-edge component used by tedge-agent for privilege elevation.
+tedge-agent spawns a `tedge-write` process when it needs to write to files that `tedge` user/group
+has no write permissions to (e.g. system files or files owned by other packages), for example when
+writing an updated configuration file as part of [`config_update` operation][1]. `tedge-agent` will
+first try to write to a file directly and only retry using `tedge-write` if direct write fails due
+to `tedge` user/group not having write permissions to either the file itself or its parent
+directory. Write permission to the parent directory is required to guarantee config updates are
+atomic.
+
+[1]: agent/tedge-configuration-management.md#handling-config-update-commands
+
+`tedge-write` will perform an atomic write, i.e. it will write to a temporary file, set file
+ownership and mode, and finally rename the temporary into the final file. If the target file already
+exists, its original ownership/mode will be preserved and optionally provided new values will be
+ignored.
+
+For `tedge-write` to work, it needs to be able to write to and set ownership and mode of all files.
+To achieve this we can use `sudo` with a `sudoers` configuration that allows `tedge` user to run
+`tedge-write` as root without password if the destination file is inside `/etc`:
+
+```sudoers title="file: /etc/sudoers.d/tedge"
+tedge   ALL = (ALL) NOPASSWD: /usr/bin/tedge-write /etc/*
+```
+
+:::note
+
+The sudoers entry will be created by thin-edge.io automatically under `/etc/sudoers.d/tedge`, but in
+non-standard setups the default configuration may need to be changed. It's necessary to provide a
+full path to `tedge-write` in the `sudoers` entry. See [`sudoers(5)`][2] for additional details.
+
+:::
+
+[2]: https://www.man7.org/linux/man-pages/man5/sudoers.5.html
+
+If `sudo.enable` is disabled, `tedge-write` will be started without `sudo`.
```

Comment:
I think it's better to show a command line example, or at least mentioning `sudo.enable` is a key of tedge config.
I would suggest one example that I came up with, but you don't have to use the same sentence at all.

```suggestion
To start `tedge-write` without `sudo`, set the tedge config `sudo.enable` to `false`.
```

```sh
sudo tedge config set sudo.enable false
```

Reply:
fixed 14fe31fc6d